src/core/core.interaction.js

```diff
@@ -21,8 +21,8 @@ function getRelativePosition(e, chart) {
 
 /**
  * Helper function to traverse all of the visible elements in the chart
- * @param chart {chart} the chart
- * @param handler {Function} the callback to execute for each visible item
+ * @param chart {Chart} the chart
+ * @param handler {function} the callback to execute for each visible item
```

Comment:
All the docs in this file switch the param type and name, should be:

```javascript
@param {Chart} chart - description ...
```

Reply:
done

Reply:
Thanks (strange, I didn't realize that I sent 2 different reviews)